src/components/Dashboard/Carousel/PreviewPost/PreviewPost.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This test does not comply with the [Guide Lines](https://github.com/FromDoppler/doppler-webapp#test).

```
import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PreviewPost from './PreviewPost';

describe('PreviewPost component', () => {
  it('should show the slide content', async () => {
    // Arrange
    const post = {
      title: 'fake title',
      description: 'fake description',
      link: 'https://localhost:3000/dashboard',
    };

    // Act
    render(<PreviewPost post={post} />);

    // Assert
    expect(screen.getByText(post.title)).toBeInTheDocument();
    expect(screen.getByText(post.description)).toBeInTheDocument();
    expect(screen.getByRole('link')).toHaveAttribute('href', post.link);
  });
});

```